### PR TITLE
Fix string formatting in qtile_top.py - issue #1017

### PR DESCRIPTION
**Pull request by frostidaho**
For issue #1017 

https://github.com/qtile/qtile/issues/1017

### CODE REVIEW

**Comment by flacjacket** [comment]
This is fine, I'm curious what it is that is causing the errors? What do the exceptions look like?

**Comment by flacjacket** [comment]
I mostly ask so when this goes in, we don't re-break it later

File: libqtile/scripts/qtile_top.py [hunk 1]
```diff
@@ -101,22 +101,22 @@ def get_stats(scr, client, group_by='lineno', limit=10, seconds=1.5,
             line = linecache.getline(frame.filename, frame.lineno).strip()
             if line:
                 code = line
-            mem = "{0:.1f} KiB".format(stat.size / 1024)
-            filename = "{0:s}:{1:s}".format(filename, frame.lineno)
-            scr.addstr(cnt + 1, 0, '{0:<3s} {1:<40s} {2:<30s}'.format(index, filename, mem))
+            mem = "{:.1f} KiB".format(stat.size / 1024.0)
```
Comment (flacjacket) [inline libqtile/scripts/qtile_top.py]: Ah, we should probably do the standard `__future__` imports for stuff like division.

**Comment by frostidaho** [comment]
It'd always raise a ValueError. e.g.,

```
In [2]: x = 37

In [3]: '{:s}'.format(x)
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-6-bb5f3dab3df6> in <module>()
----> 1 '{:s}'.format(x)

ValueError: Unknown format code 's' for object of type 'int'
```

**Comment by flacjacket** [comment]
Ah, I see, thanks!

**Pull request closed without merging**